docs-src/docs/guides/one_off_task.md

```diff
@@ -1,57 +1,49 @@
 
-This guide expands upon the [Guestbook deployment guide](/guides/guestbook) deployment guide. You must complete the procedures in that guide before you can complete the procedures listed here. After completing the procedures in the Guestbook guide, your Layer0 should contain a service named "guestbooksvc", running a deploy named "guestbook", behind a load balancer named "guestbooklb", all within an environment named "demo".
-
 ## Part 1: Prepare the task definition
 
-1. Download the [Guestbook One-off Task Definition](https://github.com/quintilesims/layer0-examples/blob/master/1offtask/Dockerrun.aws.json) and save it to your computer as **GuestbookRestore.Dockerrun.aws.json**.
-2. Edit the `GUESTBOOK_URL` environment variable for the `l0-guestbook-restore` container to the url of the loadbalancer running your guestbook application. This url can be obtained by looking at the output of the command
-
-<span style="padding-left:2em">**l0 loadbalancer get guestbooklb**</span>
-
-3. Edit the `BACKUP_FILE_URL` environment variable for the `l0-guestbook-restore` container to the url of the backup file that you wish to restore from. A sample backup file is provided at [https://github.com/quintilesims/layer0-examples/raw/master/1offtask/backup.txt](https://github.com/quintilesims/layer0-examples/raw/master/1offtask/backup.txt).
+1. Download the [Guestbook One-off Task Definition](https://github.com/quintilesims/layer0-examples/blob/master/one-off-task/Dockerrun.aws.json) and save it to your computer as **Dockerrun.aws.json**.
 
 ## Part 2: Create a deploy
-Next, you will create a new deploy for the task.
-
-**To create a new deploy:**
-
-At the command prompt, run the following command:
+Next, you will create a new deploy for the task using the **deploy create** command. At the command prompt, run the following command:
 
-<span style="padding-left:2em">**l0 deploy create GuestbookRestore.Dockerrun.aws.json guestbookrestore**</span>
+`l0 deploy create Dockerrun.aws.json one-off-task-dpl`
 
 You will see the following output:
 ```
 DEPLOY ID           DEPLOY NAME        VERSION
-guestbookrestore.1  guestbookrestore   1
+one-off-task-dpl.1  one-off-task-dpl   1
 ```
 
 ## Part 3: Create the task
-At this point, you can use the **task create** command to begin an instance of the task deployed above. This task requires two environment variables to be supplied to the `l0-guestbook-restore` container: `GUESTBOOK_URL` and `BACKUP_FILE_URL`. These were collected in Part 1 of this guide.
+At this point, you can use the **task create** command to begin an instance of the task deployed above.
 
 To run the task, use the following command:
 
-<span style="padding-left:2em">**l0 task create demo guestbookrestore guestbookrestore**</span>
+`l0 task create demo-env one-off-task-tsk one-off-task-dpl:latest`
```

Comment:
@zpatrick how would this change the output? currently v0.9 doesn't support the --wait flag so I can't accurately document what the output would look like when using the `--wait` flag. would you know which issue is tracking the change for 'task creation as a job' change? I can keep track of it and update the docs once I can confirm what the output should be.